Plugwise reorder and apply config suggestions (#36087)
home-assistant/home-assistant.github.io: source/_integrations/plugwise.markdown

 - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
 - `number` (for changing a boiler setpoint, Circlesa temperature offset)

## Configuration

{% important %}
When you have an Anna and an Adam, make sure to **only** configure the Adam integration. You can press the "IGNORE" button on the auto-discovered Anna integration to hide this integration. In case you need to rediscover the Anna integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".
{% endimportant %}

The Plugwise Smile(s) present in your network will be automatically detected via Zeroconf discovery and will be shown on the Integrations-page. All you need is the Smile ID as it's password, which is an 8 character string printed on the sticker on the bottom of you smile. To set up Plugwise when a Smile is not auto-discovered you can use this My Button:

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:

>     description: "The hostname or IP address of your Smile. For example: '192.168.1.25'. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network **. If you are looking for a different device in the Plugwise App, on the main screen first select **Gateways** -> the Smile of your choice, and then follow the previous instruction. Normally the Smile(s) are automatically discovered and you don't have to provide the hostname or IP address."

Reviewer: ```suggestion
  description: "The hostname or IP address of your Smile. For example: `192.168.1.25`. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network **. If you are looking for a different device in the Plugwise App, on the main screen first select **Gateways** -> the Smile of your choice, and then follow the previous instruction. Normally the Smile(s) are automatically discovered and you don't have to provide the hostname or IP address."
```

Username:

>     description: "Username to log in to the Smile. This should be just 'smile' - or 'stretch' for a Stretch'."

Reviewer: ```suggestion
  description: "Username to log in to the Smile. This should be just 'smile' - or 'stretch' for a Stretch'."
```

Password:

>     description: "This is the password (i.e. Smile ID) printed on the sticker on the bottom of your Smile and should be 8 characters long".

Reviewer: ```suggestion
  description: "This is the password (i.e. Smile ID) printed on the sticker on the bottom of your Smile and should be 8 characters long".
```

 - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
 - `number` (for changing a boiler setpoint, Circlesa temperature offset)

## Before configuring

{% important %}

> When you have an Anna and an Adam, make sure to **only** configure the Adam integration. You can press the "IGNORE" button on the auto-discovered Anna integration to hide this integration. In case you need to rediscover the Anna integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".

Reviewer 2: I thought this was solved in the integration? As in, it shouldn't auto discover twice?

Author: Good call, I'll double-check

Author: https://github.com/home-assistant/core/blob/4eb5734d73ee601bbd6223941ea12e8ffe0b3583/homeassistant/components/plugwise/config_flow.py#L124

It still applies though aside from discovery, I'll reword it

 - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
 - `number` (for changing a boiler setpoint, Circlesa temperature offset)

## Configuration

{% important %}
When you have an Anna and an Adam, make sure to **only** configure the Adam integration. You can press the "IGNORE" button on the auto-discovered Anna integration to hide this integration. In case you need to rediscover the Anna integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".
{% endimportant %}

The Plugwise Smile(s) present in your network will be automatically detected via Zeroconf discovery and will be shown on the Integrations-page. All you need is the Smile ID as it's password, which is an 8 character string printed on the sticker on the bottom of you smile. To set up Plugwise when a Smile is not auto-discovered you can use this My Button:

> {% include integrations/config_flow.md %}

Reviewer: ```suggestion
{% include integrations/config_flow.md %}

```

{% configuration_basic %}
Host:
    description: "The hostname or IP address of your Smile. For example: '192.168.1.25'. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network **. If you are looking for a different device in the Plugwise App, on the main screen first select **Gateways** -> the Smile of your choice, and then follow the previous instruction. Normally the Smile(s) are automatically discovered and you don't have to provide the hostname or IP address."

>     required: true
>     type: string

Reviewer: ```suggestion
```

Username:
    description: "Username to log in to the Smile. This should be just 'smile' - or 'stretch' for a Stretch'."

>     required: true
>     type: string

Reviewer: ```suggestion
```

Password:
    description: "This is the password (i.e. Smile ID) printed on the sticker on the bottom of your Smile and should be 8 characters long".

>     required: true
>     type: string

Reviewer: ```suggestion
```

 - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)

## Pre-requisites

> The Plugwise Smile(s) present in your network will be automatically discovered via Zeroconf discovery and will be shown on the Integrations-page. All you need is the Smile ID as its password, which is an 8 character string printed on the sticker on the bottom of your Smile. Repeat this for each individual Smile.

Reviewer 2: ```suggestion
The Plugwise Smile(s) in your network will be automatically discovered and shown on the integrations dashboard. All you need is the Smile ID as its password, which is an 8-character string printed on the sticker on the bottom of your Smile. Repeat this for each individual Smile.
```

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
  description: "The hostname or IP address of your Smile. For example: `192.168.1.25`. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network**. If you are looking for a different device in the Plugwise App, on the main screen first select **Gateways** -> the Smile of your choice, and then follow the previous instruction. Normally, the Smile(s) are automatically discovered, and you don't have to provide the hostname or IP address."
Username:
  description: "Username to log in to the Smile. This should be just `smile` - or `stretch` for a Stretch."
Password:
  description: "This is the password (i.e. Smile ID) printed on the sticker on the back of your Smile (i.e. Adam, Smile-T, or P1) and should be 8 characters long."
{% endconfiguration_basic %}

### Further configuration

> For a thermostat, the active schedule can be deactivated or reactivated via the climate card. Please note, that when no schedule is active, one must first be activated in the Plugwise App. Once that has been done the Plugwise Integration can manage future operations.

Reviewer 2: ```suggestion
For a thermostat, the active schedule can be deactivated or reactivated via the climate card. Please note, that when no schedule is active, one must first be activated in the Plugwise App. Once that has been done, the Plugwise Integration can manage future operations.
```

> Auto means the schedule is active, Heat means it's not active. The active thermostat schedule can be changed via the connected thermostat select-entity. Please note: that only schedules that have two or more schedule points will be shown as select options.

Reviewer 2: ```suggestion
Auto means the schedule is active, and Heat means it's not active. The active thermostat schedule can be changed via the connected thermostat select entity. Please note that only schedules with two or more schedule points will be shown as select options.
```

 - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)

> ## Before configuring

Reviewer 2: ```suggestion
## Pre-requisites
```

{% important %}
When you have an Anna and an Adam, only the Adam will be shown as discovered. Make sure to **only** configure the Adam integration, i.e. do **not** manually configure the Anna.

> {% endimportant %}

Reviewer 2: I'm still confused about this important warning... even if they tried, it will fail; so why scare the end user here?

Author: Good call ... adjusted accordingly
